packages/provider/src/tasks/tasks.ts

```diff
@@ -314,14 +315,10 @@ export class Tasks {
         dappAccount: string,
         requestHash: string,
         captchas: CaptchaSolution[],
-        signature: string // the signature to indicate ownership of account
+        signature: string, // the signature to indicate ownership of account
+        timestamp: string,
+        signedTimestamp: string
     ): Promise<DappUserSolutionResult> {
-        if (!(await this.dappIsActive(dappAccount))) {
```

Comment:
We will need to replace this with a call to the mongo client database checking that the app is active on one of our tiers. Please add as a follow up ticket.